### ISSUE TITLE
paywithgoogle Error: Uncaught (in promise): AbortError: User closed the Payment Request UI.

### ISSUE THREAD

**Issue by Granskott** [Adyen/adyen-web#1532]
We are starting to integrate google payments.
**Bug**
The payments work, however we get errors when users close the google pay sheet:

Error: Uncaught (in promise): AbortError: User closed the Payment Request UI.
(There can also be other errors, for example when the user already has another google pay session running, or when the sheet times out)
As shown in the code, we try to catch the error using onError. 

While this callback gets notified of the AbortError, the uncaught Promise error is still propagated up to the main Error handler of our Frontend.
Is there a better way to handle this promise? Our current best solution would be to tell our outmost error handler to do a string compare on error.message.

**To Reproduce**
The same seems to be true on the demo shop https://www.mystoredemo.io/#/checkout
("Pay with Google Pay" -> Hit the back arrow -> The console logs: "Uncaught (in promise) DOMException: Another PaymentRequest UI is already showing in a different tab or window." ). 
It seems that this Error is handled outside the Dropin as well.

**Simplified Code:**
script source: https://checkoutshopper-test.adyen.com/checkoutshopper/sdk/5.8.0/adyen.js

```
const checkoutConfiguration: any = {
  paymentMethodsResponse: paymentsResponse
  environment: environment,
  clientKey: clientKey,
  locale: locale,
  amount: amount,
	onSubmit: (state, dropin) => {
	  if (state.isValid) {
	    //Called when the user has successfully submited data through the google pay sheet.
	    console.log("onSubmit", state, dropin);
	  }
	},
  paymentMethodsConfiguration: {
    paywithgoogle: {
      onError: (error, component) => {
      	//Is called when user closes google pay sheet, but error still propagates
        console.error(error.name, error.message, error.stack, component);
      },
    }
  }
};

this.checkout = await AdyenCheckout(configuration);
this.dropinComponent = this.checkout.create('dropin', {
}).mount('#dropin-container');

//Some trigger
mybutton.onclick=this.dropinComponent.submit();
```

**Expected behavior**
After onError is triggered, the error should not propagate further.

Tested with Adyen web 5.0.0 , 5.8.0, 5.10.0

 - Device: Macbook Pro M1
 - OS: Mac 12.2.1
 - Browser  Chrome & others
 - Version [ 99.0.4844.51]

**Comment by strublos** [comment]
Hi @ribeiroguilherme is there something new about this issue?

**Comment by ribeiroguilherme** [comment]
This issue is fixed on the 5.27.0
